### PR TITLE
Revise users edit page

### PR DESCRIPTION
**Pull request by Robertgervais**

### CODE REVIEW

**Comment by Robertgervais** [comment]
Image 5-28-19 at 8 35 PM

**Comment by Robertgervais** [comment]
I added this vert scroll bike box (honestly I love vertical scroll now), because i was looking for something to fill that white space to the right and it seemed like the best thing.

**Comment by Robertgervais** [comment]
Image 5-29-19 at 4 50 PM


Fake user

**Comment by Robertgervais** [comment]
#803

File: app/views/admin/users/edit.html.haml [hunk 1]
```diff
@@ -24,84 +22,129 @@
...
+          %tr
+            %th Task
+            %th Completed at
+        %tbody
+          - ambassador.ambassador_task_assignments.each do |task|
+            %tr
+              %td
+                %strong= task.title
+                %br
+                = task.description_html.html_safe
```
Comment (sethherr) [inline app/views/admin/users/edit.html.haml]: I don't think you need the description. That will make things overlong.

File: app/views/admin/ambassadors/_ambassador.html.haml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+%h1= "Ambassador: #{@ambassador.username}"
```
Comment (sethherr) [inline app/views/admin/ambassadors/_ambassador.html.haml]: What's  this file for?

File: app/views/admin/users/edit.html.haml [hunk 3]
```diff
@@ -24,84 +22,129 @@
...
-      .form.form-horizontal
-        %h3
-          User attributes
-        .control-group.padded
-          %lable.control-label
+        %tr
+          %td
             Created
-          .controls.convertTime
+          %td.convertTime
```
Comment (sethherr) [inline app/views/admin/users/edit.html.haml]: Nice! I hadn't tried putting the convertTime on `td`s, it makes sense and simplifies things though!

File: app/views/admin/users/edit.html.haml [hunk 4]
```diff
@@ -24,84 +22,129 @@
...
-      .form.form-horizontal
-        .control-group
-          = f.label :confirmed, class: "control-label"
-          .controls
-            = f.check_box :confirmed, disabled: @user.confirmed
-        .control-group
-          = f.label :superuser, class: "control-label"
-          .controls
-            = f.check_box :superuser
+      = ambassador.avatar
```
Comment (sethherr) [inline app/views/admin/users/edit.html.haml]: avatar should only be included if it isn't the default image

Also, this table list can always be displayed. It isn't exclusive to ambassadors.

Comment (sethherr) [reply app/views/admin/users/edit.html.haml]: Also, this could be put into the table list

**Comment by Robertgervais** [comment]
Image 5-29-19 at 7 15 PM

**Comment by Robertgervais** [comment]
#175

**Pull request merged**